test/e2e_node/eviction_test.go

```diff
@@ -935,10 +938,12 @@ func eventuallyGetSummary(ctx context.Context) (s *kubeletstatsv1alpha1.Summary)
 
 // returns a pod that does not use any resources
 func innocentPod() *v1.Pod {
+	var gracePeriod int64 = 1
```

Comment:
https://github.com/kubernetes/kubernetes/pull/119570 gave me a lead on a possible failure for eviction.  

In case of hard eviction, we were waiting for 30 seconds which could account for flakiness.  

Not sure if this will fix all the issues but it will at least starting terminating.  

I was seeing problems where pods were not being terminated in the timeframe we were waiting.  This at least starts eviction.

Reply:
interesting. Can you add a comment in code explaining the need for this?

Reply:
done.